### ISSUE TITLE
Default Adobe TypeKit configuration

### ISSUE THREAD

**Issue by samuelpowell** [JuliaLang/JuliaBox#273]
By default, the JuliaBox `engineinteractive` container will render `index.tpl` which will attempt to pull the Adobe TypeKit generated `//use.typekit.net/cpz5ogz.js`. This will only work on the domains for which this TypeKit has been enabled (I assume only `juliabox.org`).

I would like to document this, ideally guiding the user to set up their own TypeKit configuration to match whatever is set up for `juliabox.org`, as a starting point. Can some with access let me know what this configuration is?

**Comment by ViralBShah** [comment]
The easiest thing would be for the default not to use TypeKit. Also, the logos at the bottom are only for juliabox.org, and not necessarily for the self-installed version.

**Comment by samuelpowell** [comment]
Yes, one would hope that no-one installs and *publicly* serves JuliaBox as it is configured when they pull it from this repository.

I suppose one option would be to include a very simply `index.tpl` which is the default, and rename the JuliaBox.org version. This would probably make the least work for @tanmaykm, whilst avoiding this problem.

Any thoughts welcomed.

**Comment by tanmaykm** [comment]
To let non JuliaBox installations have full control of the UI, we should ideally allow replacing all the templates. I'm not sure if we'll need to do something similar for static files being served through nginx. People will want some tweaking for it to look good without typekit fonts.

How about the following options:

1. Switch the folder altogether. Simpler. But may have unnecessary duplication.
2. Look into a `custom_templates` folder and overwrite only the stuff present there.

My preference is for 1, because it is simpler and gives maximum flexibility.

**Comment by ViralBShah** [comment]
It already looks quite decent without typekit.

**Comment by ViralBShah** [comment]
Cc: @shashi
